### PR TITLE
1

### PR DESCRIPTION
**Pull request by serhii-dvorets**
https://serhii-dvorets.github.io/react_pagination/

### CODE REVIEW

File: src/Components/Pagination.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  componentDidMount() {
+    const array = this.fillTotal();
+
+    this.setState({ total: array });
+  }
+
+  fillTotal = () => {
+    const arrayOfItems = [];
+
+    for (let i = 1; i <= 42; i += 1) {
```
Comment (vitaliy-martynyuk) [inline src/Components/Pagination.tsx:31]:
```suggestion
    for (let i = 1; i <= 42; i ++) {
```

File: src/Components/PageNumbers.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+};
+
+export const PageNumbers: React.FC<Props> = ({
+  pages,
+  setPage,
+  setPrevPage,
+  setNextPage,
+  currentPage,
+}) => {
+  const renderButton = (button: number) => {
```
Comment (vitaliy-martynyuk) [inline src/Components/PageNumbers.tsx:16]: your code is not DRY in this file. try using conditional rendering

**Review by vitaliy-martynyuk — CHANGES_REQUESTED** [review]

File: src/Components/Pagination.tsx (hunk 1 above)
Comment (serhii-dvorets) [reply src/Components/Pagination.tsx:31]: ESLint doesn't allow such short form. It returns "Unary operator '++' used"

**Review by vitaliy-martynyuk — APPROVED** [review]
GJ!

File: src/Components/Pagination.tsx (hunk 1 above)
Comment (vitaliy-martynyuk) [reply src/Components/Pagination.tsx:31]: that's strange. I'll check ESLint config for this task

**Pull request closed without merging**